tfprotov6/resource.go

```diff
@@ -606,7 +606,10 @@ type MoveResourceStateRequest struct {
 	TargetTypeName string
 
 	// SourceIdentity is the identity of the source resource.
-	SourceIdentity *ResourceIdentityData
+	SourceIdentity *RawState
```

Comment:
Similar comment to protov5

```suggestion
	//
	// Only the underlying JSON field is populated.
	SourceIdentity *RawState
```